app.json

```diff
@@ -1,63 +1,12 @@
 {
   "name": "intranet",
   "scripts": {
-    "postdeploy": "python manage.py runserver"
+    "postdeploy": "python manage.py deploy"
   },
   "env": {
```

Comment:
@johnyu95 - Any other default environment variables we need?

Reply:
Not that I can think of.